src/datasources/workspace/WorkspaceDataSource.ts

```diff
@@ -14,26 +15,36 @@ export class WorkspaceDataSource extends DataSourceBase<WorkspaceQuery> {
 
   baseUrl = this.instanceSettings.url + '/niuser/v1';
 
-  defaultQuery = {
-    constant: 3.14,
-  };
-
-  async runQuery(query: WorkspaceQuery, { range }: DataQueryRequest): Promise<DataFrameDTO> {
-    return {
-      refId: query.refId,
-      fields: [
-        { name: 'Time', values: [range.from.valueOf(), range.to.valueOf()], type: FieldType.time },
-        { name: 'Value', values: [query.constant, query.constant], type: FieldType.number },
-      ],
-    };
+  defaultQuery = {};
+
+  async runQuery(_query: WorkspaceQuery): Promise<DataFrameDTO> {
+    return { fields: this.workspacesToFields( await this.getWorkspaces()) };
```

Comment:
I think we had talked about having this just list workspace names instead of both the id and name. 
Admittedly, the usefulness of this data source in a panel is really low, so it probably doesn't matter, but I think if someone does use it they wouldn't care about the ids.